test/integration/replication.bats

```diff
@@ -2,8 +2,8 @@
 
 load helpers
 
-# Address on which the store will listen (random port between 8000 and 9000).
-STORE_HOST=127.0.0.1:$(( ( RANDOM % 1000 )  + 8000 ))
+# Address on which the store will listen
+STORE_HOST=127.0.0.1:8500
```

Comment:
I was thinking about abstracting the test for the 3 supported backends because it's just Consul for now so I'll make sure I add this enhancement later on :+1: 

I fixed the port on this one because I shut down the store and bring it back, the swarm Manager expects the same endpoint to be reachable.